src/containerapp/azext_containerapp/tests/latest/test_containerapp_maintenance_config.py

```diff
@@ -0,0 +1,68 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+
+from azure.cli.command_modules.containerapp._utils import format_location
+from azure.mgmt.core.tools import parse_resource_id
+
+from azure.cli.testsdk.scenario_tests import AllowLargeResponse
+from azure.cli.testsdk import (ScenarioTest, ResourceGroupPreparer, JMESPathCheck)
+from .utils import create_containerapp_env, prepare_containerapp_env_for_app_e2e_tests
+
+TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
+
+from .common import TEST_LOCATION, STAGE_LOCATION
+from .utils import prepare_containerapp_env_for_app_e2e_tests
+
+
+class ContainerAppMaintenanceConfigTest(ScenarioTest):
+    @AllowLargeResponse(8192)
+    @ResourceGroupPreparer(location="eastus")
+    def test_containerapp_maintenanceconfig_crudoperations_e2e(self, resource_group):
+        
+        self.cmd('configure --defaults location={}'.format(TEST_LOCATION))
+        
+        
+        env_name = self.create_random_name(prefix='aca-maintenance-config-env', length=30)
+        self.cmd('containerapp env create -g {} -n {} --location {}  --logs-destination none --enable-workload-profiles'.format(resource_group, env_name, TEST_LOCATION))
+
+        duration = 10
+        weekday = "Sunday"
+        startHour = 12
+
+        # create a container app environment for a Container App Maintenance Config resource
+        self.cmd('containerapp env show -n {} -g {}'.format(env_name, resource_group), checks=[
+            JMESPathCheck('name', env_name)
+        ])
+
+        ## test for CRUD operations on Maintenance Config
+        self.cmd("az containerapp env maintenance-config add --resource-group {} --environment {} -d {} -w {} -s {}".format(resource_group, env_name, duration, weekday, startHour))
+
+        # verify the resource
+        self.cmd("az containerapp env maintenance-config show --resource-group {} --environment {}".format(resource_group, env_name), checks=[
+            JMESPathCheck('properties.scheduledEntries[0].durationHours', duration),
+            JMESPathCheck('properties.scheduledEntries[0].startHourUtc', startHour),
+            JMESPathCheck('properties.scheduledEntries[0].weekDay', weekday),
+        ])
+
+        updatedDuration = 11
+        updatedWeekday = "Tuesday"
+
+        # update the MaintenanceConfig, check duration and weekday are updated and start hour remains the same
+        self.cmd("az containerapp env maintenance-config update --resource-group {} --environment {} -d {} -w {}".format(resource_group, env_name, updatedDuration, updatedWeekday), checks=[
```

Comment:
You can add a test to update with nothing:
For example:
`az containerapp env  maintenance-config update --environment env-francecentral -g client.env_rg_francecentral`